plugins/filter/jc.py

```diff
@@ -94,7 +94,11 @@ def jc(data, parser, quiet=True, raw=False):
         dictionary or list of dictionaries
 
     Example:
-
+        - name: add the jc prereqs to the running machine
```

Comment:
Maybe add a comment above saying this section is an optional check that the jc library is installed on the Ansible Controller?